src/services/collaborativesearch.service.ts

```diff
@@ -662,6 +675,42 @@ export class CollaborativesearchService {
         }
     }
 
+    private resolveButNotWithAbort(projection: [projType.aggregate, Array<Aggregation>]
+        | [projType.search, Search]
+        | [projType.geoaggregate, Array<Aggregation>]
+        | [projType.geohashgeoaggregate, GeohashAggregation]
+        | [projType.geosearch, Search]
+        | [projType.tiledgeosearch, TiledSearch]
+        | [projType.count, Count]
+        | [projType.range, RangeRequest]
+        | [projType.compute, ComputationRequest], collaborations: Map<string, Collaboration>, abortableSignal,
+        contributorId?: string, filter?: Filter, isFlat?: boolean, max_age = this.max_age
+    ): Observable<any> {
+        try {
+            const filters: Array<Filter> = new Array<Filter>();
+            if (contributorId) {
+                collaborations.forEach((k, v) => {
+                    if (v !== contributorId && k.enabled) {
+                        if (k.filter) { filters.push(k.filter); }
+                    } else {
+                        return;
+                    }
+                });
+            } else {
+                collaborations.forEach((k, v) => {
```

Comment:
Same question

Reply:
this code was duplicated and I removed it. This is still in the code though. I prefer to treat it in another PR, to avoid unexpected regression